syz-fuzzer/fuzzer.go

```diff
@@ -288,11 +254,16 @@ func main() {
 		fuzzerTool.procs = append(fuzzerTool.procs, proc)
 		go proc.loop()
 	}
-	// Start send input workers.
-	for i := 0; i < *flagProcs*2; i++ {
-		go fuzzerTool.sendInputsWorker(fuzzerObj.Config.NewInputs)
+	// Query enough inputs at the beginning.
+	fuzzerTool.exchangeDataCall(inputsCount, nil)
+
+	// Set up communication workers.
+	for i := 0; i < 2; i++ {
```

Comment:
I've added the stat. So far, on a setup with 10 qemu VMs:

One communication worker per VM gives ~120 blocks/minute (in sum).
Two communication workers per VM gives ~75 blocks/minute (in sum).

Both don't seem to be significant, but I'm curious what it would look like on GCE.